app/components/request_list.tsx

```diff
@@ -108,10 +108,9 @@ const RequestList = (props: RequestObj) => {
       pageSize={props.PageDetails[props.tabId] ? props.PageDetails[props.tabId].currentRowSize : 10}
       onPageChange={changedPageNo => props.handlePaginationChange(changedPageNo, props.tabId, "currentPageNumber")}
       onPageSizeChange={changedRowSize => props.handlePaginationChange(changedRowSize, props.tabId, "currentRowSize")}
-      freezeWhenExpanded={true}
+      collapseOnDataChange={false}
```

Comment:
why we need to pass this `collapseOnDataChange`? since its false, we can skip right?

Reply:
No. The default value is set to true. check [this](https://github.com/react-tools/react-table#props)